contracts/modules/governance/Governance.sol

```diff
@@ -865,8 +865,11 @@ contract Governance is IGovernance, LegacyMasterAware {
           stakingPool.manager() == msg.sender,
           "One or more staking pools are not managed by the sender"
         );
-        uint activeStake = stakingPool.activeStake();
-        voteWeight = voteWeight.add(activeStake);
+        (
+          uint stakingRewards,
+          uint stakingDeposits
+        ) = tokenInstance.stakingPoolNXMBalances(managedStakingPoolIds[i]);
+        voteWeight = voteWeight.add(stakingRewards).add(stakingDeposits);

```

Comment:
Rewards are not supposed to be used because they can be withdrawn at any time regardless of the voting lock.

Reply:
Not anymore, this was changed today. The `withdraw` function should always revert while the manager is locked.

Reply:
On a second thought I see no reason why we wouldn't allow reward withdrawals since rewards are negligible given the 15% voting power cap, so never mind. See the commit below.